### DISCUSSION TITLE
Add support for media queries

### DISCUSSION THREAD

**Discussion by vallemar** [NativeScript/rfcs#46] [1. Ideas & Discussions]
I think it would be interesting to add media query support to NativeScript. I don't think as a first iteration all the standardization is needed but the basic uses of width, height and orientation are needed:

```css
@media (width: 360) {
}
@media (min-width: 35) {
}
@media (max-width: 50) {
}


@media (height: 360) {
}
@media (min-height: 25) {
}
@media (max-height: 40) {
}

@media (orientation: landscape) {
}
@media (orientation: portrait) {
}
```

It would contribute a lot when designing views and I think it would be compatible with tailwind and its responsive logic.

**Comment by rigor789** [comment]
https://github.com/NativeScript/NativeScript/pull/10530
